src/lib_ccx/ccx_encoders_ssa.c

```diff
@@ -168,11 +187,18 @@ int write_cc_buffer_as_ssa(struct eia608_screen *data, struct encoder_ctx *conte
 	unsigned h1, m1, s1, ms1;
 	unsigned h2, m2, s2, ms2;
 	int wrote_something = 0;
+	int used_row_count = 0;
 
 	int prev_line_start = -1, prev_line_end = -1;	    // Column in which the previous line started and ended, for autodash
 	int prev_line_center1 = -1, prev_line_center2 = -1; // Center column of previous line text
 	int empty_buf = 1;
-	for (int i = 0; i < 15; i++)
+
+	int first_row = -1;
+	int x, y;
+	char pos_tag[64];
+	int i;
```

Comment:
Why move the loop counter outside each loop, is there a reason for this old school style?